.travis.yml

```diff
@@ -149,7 +149,7 @@ matrix:
   - os: osx
     name: Python 3.7, c++14, AppleClang 9, Debug build
```

Comment:
Description is still 3.7

Reply:
How did I miss that? I'll have to checkout & force push now...

Reply:
I can do it in a few hours as well!

Reply:
Done.